QuizUp/actions/update_translations.rb

```diff
@@ -0,0 +1,40 @@
+module Fastlane
+  module Actions
+    class UpdateTranslationsAction < Action
+      def self.run(params)
+
+        command = [
+          "uhura pull --yes"
+        ].join(' ')
+        
+        Actions.sh command
+        
+        begin
+          Actions.sh("git add -u")
+          Actions.sh("git commit -m 'Updated translations'")
+          rescue
+            # In case there are no new translations.
+          end
+          
+        end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Download the latest translations using uhura"
```

Comment:
Its an internal python tool that downloads and install translations file from OneSky